getting-started-tutorial/tutorial-2-Model-training.ipynb

```diff
@@ -0,0 +1,2021 @@
+{
```

Comment:
<p>Although this is a tutorial, <code>pickle</code>  seems to have a lot of problems with different versions.</p><p>It will be safer to use <code>cloudpickle</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>